pallets/creditcoin/src/types/loan_terms.rs

```diff
@@ -56,19 +58,20 @@ pub struct InterestRate {
 #[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
-pub struct LoanTerms {
+pub struct LoanTerms<Hash> {
```

Comment:
Will it ever be a case where a lender/borrower want a specific transferkind?

Reply:
Hmmm not sure, what do you think @jfrank-summit?

Reply:
I don't think so, the goal will be the transfer, so any supported transfer kind on a currency should suffice.